src/util/constants.rs

```diff
@@ -35,3 +35,9 @@ pub const ADDITIONAL_METADATA_KEY: &str = "asset_additional_metadata";
 /// A constant declaration to ensure the word "nhash" does not have typos when used throughout the
 /// contract's source.
 pub const NHASH: &str = "nhash";
+/// A constant declaration to ensure the word "usd" does not have typos when used throughout the
+/// contract's source.
+pub const USD: &str = "usd";
```

Comment:
Can probably just remove this one now that's not gonna be used :rip: :pour-one-out:

Reply:
done been removed